plugins/services/src/js/components/__tests__/DeploymentsModal-test.js

```diff
@@ -0,0 +1,31 @@
+import Item from "#SRC/js/structs/Item";
+/* eslint-disable no-unused-vars */
+const React = require("react");
+/* eslint-enable no-unused-vars */
+
+const DeploymentsModal = require("../DeploymentsModal").WrappedComponent;
+const Application = require("../../structs/Application");
+
+describe("DeploymentsModal", function() {
+  describe("#renderStatus", function() {
```

Comment:
This feels like you're testing some private implementation detail of this component... In general I feel it would be nice to either extract renderStatus and test that, or test the rendering of the entire component